rfc/accepted/0001-new-rfc-process/index.md

```diff
@@ -0,0 +1,22 @@
+<!--
+Copyright (c) The OpenTofu Authors
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# RFC-0001: New RFC process
+
+## Abstract
+
+This RFC adds a new RFC process that enables better conversations and longer term collaboration between the core team and the community.
```

Comment:
Suggestion:

The RFC process is introduced to facilitate long term discussions and complex planning among the core team and the community.  It is a collaborative planning space where features and sweeping changes can be hashed out in depth.